S/SHTns/build_tarballs.jl

```diff
@@ -2,53 +2,66 @@
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
 
-include(joinpath(@__DIR__, "..", "..", "platforms", "microarchitectures.jl"))
+YGGDRASILPATH = joinpath(@__DIR__, "..", "..")
+
+include(joinpath(YGGDRASILPATH, "fancy_toys.jl"))
+include(joinpath(YGGDRASILPATH, "platforms", "microarchitectures.jl"))
+include(joinpath(YGGDRASILPATH, "platforms", "cuda.jl"))
 
 name = "SHTns"
-version = v"3.6.6"
+version = v"3.7"
+version_string = version.patch == 0 ? string(version.major)*"."*string(version.minor) : string(version)
```

Comment:
This seems a lot simpler:
```suggestion
version_string = "3.7"
version = VersionNumber(version_string)
```

Reply:
true 👍